content/How to work with Stripe.md

```diff
@@ -0,0 +1,133 @@
+# **Online payments made SIMPLE - How to work with Stripe**
+
+![main image](images/payment.jpg)
+
+Here at [Rootstrap](https://www.rootstrap.com/mobile-app-development-los-angeles/), we care about not reinventing the wheel and making things simple (along with high quality of course).
+
+In this blog post, you’ll learn how to start working with Stripe and quickly have fully functioning online payments in your apps.
+
+
+## 1) WHY STRIPE?
+
+Pros
+
+ - Easy to implement and use
+
+ - Fast to develop, so your client will be happy
+
+ - Solves most of your usual payment problems, so you don't lose time or clients (even worst)
+
+ - Amazing dashboard with a lot of capabilities so your clients financial team can work along with you
+
+Cons
+
+ - Expensive (high % fee)
+
+## 2) INSTALLATION
+
+This post assumes you already created a Stripe account and so you have access to dashboard and its configuration.
+
+#### RAILS
+
+- Add these two gems:
+    - [Stripe](https://github.com/stripe/stripe-ruby) to achieve the integration
+    - [Stripe Testing](https://github.com/stripe-ruby-mock/stripe-ruby-mock) to test your integration, you don't want to end up writing lots of mocking classes, right?
+- Configure your keys & version from the Stripe dashboard
+
+https://gist.github.com/fedeagripa/0ae76f1f2a5b77f0a12fdbc2eeaa5904
+
+#### REACT
+
+- Add this package [Stripe](https://github.com/stripe/react-stripe-elements)
+- Configure your App to use the same api key as for rails (make sure it is the same, as you start moving between envs you may forget it). Remember that there is a testing key and a live one.
+
+Add an env file to store your keys
+https://gist.github.com/fedeagripa/5f99c824308ba899d6f7deb5fb85ff76
+
+Add your Stripe wrapper
+
+https://gist.github.com/fedeagripa/f2f64556a820abf9b377fa16f256d0d2
+
+## 3) START USING PAYMENTS WITH STRIPE
+
+### CREDIT CARDS
+
+##### REACT - DO YOURSELF A FAVOR AND USE THE EXISTING COMPONENT
+
+I'm not a fan of reinventing the wheel by any means, the design this components provide are more than enough for 99% of the apps you will be building. But if you insist, be prepared to spend 2 weeks dealing with details instead of 2 days.
```

Comment:
```suggestion
I'm not a fan of reinventing the wheel by any means, the design these components provide is more than enough for 99% of the apps you will be building. But if you insist, be prepared to spend 2 weeks dealing with details instead of 2 days.
```